Documentation/packaging.md

```diff
@@ -30,44 +29,19 @@ patch, bc
-#### Additional run-time dependencies for the host flavor
-
-- systemd >= v222 with systemctl, systemd-shutdown, systemd, systemd-journald.
-- bash
-
-## Ownership and permissions of rkt directories
+### Offline builds
 
-In general, subdirectories of `/var/lib/rkt` should be created with the same ownership and permissions as done by `rkt install`, see [directory list](https://github.com/coreos/rkt/blob/master/rkt/install.go#L44).
+By default, the rkt build will download a CoreOS PXE image from the internet and extract some binaries, such as `systemd-nspawn` and `bash`. However, some packaging environments don't allow internet access during the build. To work around this, download the CoreOS PXE image before starting the build process, and use the `--with-coreos-local-pxe-image-path` and `--with-coreos-local-pxe-image-systemd-version` parameters. For more details, see the [configure script parameters documentation][build-config].
 
-The package should create a group `rkt` and `/var/lib/rkt` should belong to group `rkt` with the `setgid` bit set (`chmod g+s`)
+### Bundling with systemd
 
-When the ownership and permissions of `/var/lib/rkt` are set up correctly, users member of `rkt` should be able to fetch ACIs without using sudo.
-However, `root` is still required to run pods.
-
-## Offline builds
-
-By default, rkt compilation will download a CoreOS PXE image from internet and extract some binaries such as `systemd-nspawn` and `bash`.
-However, some packaging environments don't allow internet access during the build.
-
-To solve this, you can download the CoreOS PXE image before starting the build process and use the `--with-coreos-local-pxe-image-path` and `--with-coreos-local-pxe-image-systemd-version` parameters.
-For more details, see [configure script parameters documentation](build-configure.md).
-
-## Bundling with systemd
-
-Most Linux distributions don't allow the use of prebuilt binaries, or reuse of code that is already otherwise packaged.
-systemd falls in this category, as Debian and Fedora already package systemd and rkt needs systemd.
+Most Linux distributions don't allow the use of prebuilt binaries, or reuse of code that is already otherwise packaged. systemd falls in this category, as Debian and Fedora already package systemd and rkt needs systemd.
 
 - [Debian Policy Manual, section 4.13 Convenience copies of code](https://www.debian.org/doc/debian-policy/ch-source.html#s-embeddedfiles)
 - [Fedora Packaging Guidelines](https://fedoraproject.org/wiki/Packaging:Guidelines#No_inclusion_of_pre-built_binaries_or_libraries)
 - [Fedora Packaging Committee](https://www.mail-archive.com/devel@lists.fedoraproject.org/msg88276.html)
 
-In order to avoid build-dependency on systemd in rkt, a build option was added to build stage1 without including binaries that are build from external sources.
+In order to avoid rkt's build dependency on systemd, a build option was added to build stage1 without including binaries from external sources.
```

Comment:
shouldn't we tweak the tenses here now? "a build option was added" feels like a changelog..